doc/source/tracking/icepyx_pubs.bib

```diff
@@ -0,0 +1,41 @@
+@misc{2020_IS2-HW-tutorials,
+  author       = {Arendt, Anthony and 
+  		 Scheick, Jessica and 
+                  Shean, David and
+                  Buckley, Ellen and
+                  Grigsby, Shane and
+                  Haley, Charley and
+                  Heagy, Lindsey and
+                  Mohajerani, Yara and
+                  Neumann, Tom and
+                  Nilsson, Johan and
+                  Markus, Thorsten and
+                  Paolo, Fernando S. and
+                  Perez, Fernando and
+                  Petty, Alek and
+                  Schweiger, Axel and
+                  Smith, Benjamin and
+                  Steiker, Amy and
+                  Alvis, Sebastian and
+                  Henderson, Scott and
+                  Holschuh, Nick and
+                  Liu, Zheng and
+                  Sutterly, Tyler},
+  title        = {2020 {ICESat-2 Hackweek Tutorials}},
+  month        = aug,
+  year         = 2020,
+  publisher    = {Zenodo},
+  version      = {1.0.0},
+  doi          = {10.5281/zenodo.3966463},
+  url          = {https://doi.org/10.5281/zenodo.3966463},
+ comment	= {tutorials}
+}
+
+@misc{js2019agu,
```

Comment:
here, I am wondering if it would be worth creating a zenodo archive of the poster and using that doi? that way the poster itself is a bit more discoverable

Reply:
Is the ESSOAR archive not open access, or just not as discoverable as Zenodo? I think discoverability is a good point, though I wonder if at this point it just makes the most sense to plan on doing this with this year's AGU presentation.